.github/workflows/pr-verify.yml

```diff
@@ -21,4 +21,4 @@ jobs:
     - name: Build
       run: mvn -B clean install -DskipTests
     - name: Run tests
-      run: mvn -B verify --file pom.xml
```

Comment:
why do we need to specify "--file pom.xml" btw?

Reply:
I'm not sure it's necessary - I just took this from the default/example maven workflow.